logic/tool/TypeResolver.java

```diff
@@ -68,6 +70,19 @@ public TypeResolver(ConceptManager conceptMgr, TraversalEngine traversalEng, Log
         this.logicCache = logicCache;
     }
 
+    public ResourceIterator<Map<Reference.Name, Label>> retrievePossibleTypeCombos(Conjunction conjunction, boolean insertable) {
```

Comment:
i think we can just call this `combinations(Conjunction, boolean)`

Reply:
Changed.